### PR TITLE
add onMessage props,  postMessage  methods

### PR DESCRIPTION
**Pull request by qingyangmoke**

### CODE REVIEW

File: src/native-common/WebView.tsx [hunk 1]
```diff
@@ -65,6 +66,13 @@ export class WebView extends RX.WebView<{}> {
             webView.goForward();
         }
     }
+    
+    postMessage(message: string, targetOrigin?: string) {
+        const webView : RN.WebView = this.refs[WEBVIEW_REF] as RN.WebView;
+        if (webView) {
+            webView.postMessage(message,targetOrigin);
```
Comment (erictraut) [inline src/native-common/WebView.tsx]: Please add space after comma for coding consistency.

File: src/native-common/WebView.tsx [hunk 2]
```diff
@@ -41,6 +41,7 @@ export class WebView extends RX.WebView<{}> {
                 scalesPageToFit={ this.props.scalesPageToFit }
                 onError={ this.props.onError }
                 onLoadStart={ this.props.onLoadStart }
+                onMessage={this.props.onMessage}
```
Comment (erictraut) [inline src/native-common/WebView.tsx]: Please add spaces within the curly brackets to match other code.

Comment (erictraut) [inline src/native-common/WebView.tsx]: This prop is not defined on the WebViewProps object within Types.ts. It's also not clear how this same prop would be supported in the web implementation. In general, all props that are supported in ReactXP should be implemented in all platforms. All props should also be documented in the components/docs directory (in this case, in the components/docs/webview.md file).

**Comment by ms-deregtd** [comment]
onMessage/postMessage is bidirectional on both web and webviews.  The inner and outer sites need to ensure that their origins match for it to be bidirectional, but it's quite doable.  onMessage should be fine on web unless I'm missing something.

File: src/native-common/WebView.tsx (hunk 1 above)
Comment (qingyangmoke) [inline src/native-common/WebView.tsx]: sorry  ,  updated

**Comment by Possum8** [comment]
THANKS

File: src/native-common/WebView.tsx (hunk 1 above)
Comment (erictraut) [inline src/native-common/WebView.tsx]: There's still no space after the comma. Should be "message, targetOrigin".

File: src/native-common/WebView.tsx [hunk 3]
```diff
@@ -41,6 +41,7 @@ export class WebView extends RX.WebView<{}> {
                 scalesPageToFit={ this.props.scalesPageToFit }
                 onError={ this.props.onError }
                 onLoadStart={ this.props.onLoadStart }
+                onMessage={ this.props.onMessage }
```
Comment (erictraut) [inline src/native-common/WebView.tsx]: If we're going to support onMessage, then this needs to be added to WebViewProps in Types.ts. And a corresponding implementation is needed for the web implementation of WebView.tsx.

**Pull request closed without merging**